Testing/SystemTests/tests/analysis/LoadAndCheckBase.py

```diff
@@ -1,6 +1,6 @@
 #pylint: disable=no-init,invalid-name
 """
-These system tests are to verify the behaviour of the ISIS reflectometry reduction scripts
+These system tests are to verify the behaviour of the ISIS Reflectometry (Old) reduction scripts
```

Comment:
I think in this case we shouldn't replace "ISIS Reflectometry" with "ISIS Reflectometry (Old)" as we are referring to ISIS reflectometry scripts, not to any of the interfaces. The same applies to some of the occurrences below.